src/main/java/com/google/firebase/auth/AbstractFirebaseAuth.java

```diff
@@ -939,13 +939,15 @@ protected String execute() throws FirebaseAuthException {
   }
 
   /**
-   * Creates a new provider OIDC Auth config with the attributes contained in the specified {@link
-   * OidcProviderConfig.CreateRequest}.
+   * Creates a new provider OIDC Auth provider config with the attributes contained in the specified
```

Comment:
This still reads a bit weird. Shouldn't it be `a new OIDC auth provider config...`?

Reply:
Sorry, thanks for catching that typo.